packages/dds/tree/src/test/cursorTestSuite.ts

```diff
@@ -13,51 +13,50 @@ import {
 	type ITreeCursor,
 	type JsonableTree,
 	type PathRootPrefix,
+	type TreeNodeSchemaIdentifier,
 	type UpPath,
 	compareFieldUpPaths,
 	rootFieldKey,
 	setGenericTreeField,
 } from "../core/index.js";
 import { leaf } from "../domains/index.js";
 import {
-	Any,
-	FieldKinds,
-	FlexFieldSchema,
-	SchemaBuilderBase,
 	cursorForJsonableTreeNode,
 	jsonableTreeFromCursor,
 	prefixFieldPath,
 	prefixPath,
 } from "../feature-libraries/index.js";
 import { brand } from "../util/index.js";
 
-import { expectEqualFieldPaths, expectEqualPaths } from "./utils.js";
-
-const schemaBuilder = new SchemaBuilderBase(FieldKinds.required, {
-	scope: "Cursor Test Suite",
-	libraries: [leaf.library],
-});
-
-export const emptySchema = schemaBuilder.object("Empty object", {});
-const emptySchema2 = schemaBuilder.object("Empty object 2", {});
-const emptySchema3 = schemaBuilder.object("Empty object 3", {});
-export const mapSchema = schemaBuilder.map(
-	"Map",
-	FlexFieldSchema.create(FieldKinds.sequence, [Any]),
-);
-// object with fixed shape
-export const objectSchema = schemaBuilder.object("object", {
-	child: leaf.number,
-});
-export const identifierSchema = schemaBuilder.object("identifier-object", {
-	identifier: FlexFieldSchema.create(FieldKinds.identifier, [leaf.string]),
-});
-export const testTreeSchema = schemaBuilder.intoSchema(
-	FlexFieldSchema.create(FieldKinds.sequence, [Any]),
-);
+import {
+	expectEqualFieldPaths,
+	expectEqualPaths,
+	IdentifierSchema,
+	JsonArray,
+	jsonObjectSchemaIdentifier,
+} from "./utils.js";
+import { SchemaFactory } from "../simple-tree/index.js";
+
+const sf = new SchemaFactory("Cursor Test Suite");
+
+export const EmptyObject = sf.object("empty object", {});
```

Comment:
Unless you specifically want to use POJO mode, its preferred to use the "Customizable" mode. See the doc comment on SchemaFactory for details.